content/blog/status-update-39-20.md

```diff
@@ -0,0 +1,117 @@
+<figcaption>Pic. 2: Message payload presentation</figcaption>
+</figure> 
+</center>
+</div>
+<div class="column" style="float: right;width: 50%;">
+<center>
+<figure>
+<img src="/images/posts/status-update-39/pic3-circ-example.webp" alt="Generated payload example" style="width:50%">
+<figcaption>Pic. 3: Generated payload example</figcaption>
+</figure>
+</center>
+</div>
+</div> 
+
+## Generate HTML Docs into a Single File
+
+Thanks to the contribution from [Gordeev Artem](https://github.com/GordeevArt), you can now generate HTML documentation into a single file. Like what?
+
+To generate HTML documentation for AsyncAPI files, you can use our [HTML generator template](https://github.com/asyncapi/html-template). It generates an index.html file with correctly rendered content of the AsyncAPI file. In addition, it also references additional files necessary for nice display of the HTML, JavaScript, and CSS files. 
+
+From time to time, the community asked that it would be nice if the template could generate only one index.html file with all the JavaScript and CSS inline inside the HTML file.
+
+Now it is possible! There is a new parameter added to the template called singlePage. Just pass it for example, in the CLI like this -p singlePage=true.
+
+## Parser with More Helpers and Better API docs
+
+We released [Parser release candidate 5](https://github.com/asyncapi/parser-js/releases/tag/v1.0.0-rc.5) with features you want to have in a library before you announce the 1.0.0 release. Thanks to generous support from our community member [Maciej Urbanczyk](https://github.com/magicmatatjahu), this release candidate contains the following additions:
+Missing externalDocs field in AsyncAPIDocument model
+New functions to all models that need it: hasBindings, bindingProtocols, hasBinding(name), binding(name), extensionKeys, extKeys, hasExtension, hasExt, tagNames, tag, hasTag, hasDescription
+Much better API documentation. In the past, it was not only missing functions that were not available but also functions that were not documented because of the wrong usage of JSDocs. Now see on your own how significant is the change by looking just on the list of functions available for ChannelParameter model:
+
+        
+<div class="row">
+<div class="column" class="column" style="float: left;width: 50%;">
+<center>
+<figure>
+<img src="/images/posts/status-update-39/pic1-api-after.webp" alt="ChannelParameter Docs after release" style="width:70%">
+<figcaption>Pic. 1: ChannelParameter Docs after release</figcaption>
+</figure> 
+</center>
+</div>
+<div class="column" style="float: right;width: 50%;">
+<center>
+<figure>
+<img src="/images/posts/status-update-39/pic2-api-before.webp" alt="ChannelParameter Docs before release" style="width:40%">
+<figcaption>Pic. 2: ChannelParameter Docs before release</figcaption>
+</figure>
+</center>
+</div>
+</div>
```

Comment:
It would be great to change the order of these two images, showing "before" first and "after" second.

Also, I'd not use inline styles here. It breaks mobile devices. If you want to use this kind of composition, better if you register a [custom shortcode](https://github.com/asyncapi/website#shortcodes).